install.sh

```diff
@@ -904,7 +904,7 @@ TasksMax=infinity
 TimeoutStartSec=0
 Restart=always
 RestartSec=5s
-ExecStartPre=/bin/sh -xc '! /usr/bin/systemctl is-enabled --quiet nm-cloud-setup.service'
```

Comment:
Why did you drop the `--quiet`? Without this the command outputs "enabled" to stdout when the service is enabled. We don't need any output, we just need the exit status.

```console
brandond@dev01:~$ /usr/bin/systemctl is-enabled nm-cloud-setup.service 2>/dev/null
enabled
brandond@dev01:~$
```

Reply:
That was a mistake. I was testing and probably forgot, sorry!